.taskcluster.yml

```diff
@@ -251,6 +251,13 @@ tasks:
                         cron: {$json: {$eval: 'cron'}}
                         treeherder:
                           symbol: raptor-D
+                        notify:
+                          email:
+                            link:
+                              text: Treeherder Job
+                              href: "https://treeherder.mozilla.org/#/jobs?repo=${project_name}&revision=${head_rev}"
+                            subject: "[${project)name}] Raptor decision task failed"
```

Comment:
I think this is a typo, and should be `[${project_name}]`

Reply:
Oops, done.